app/assets/javascript/app/actions/fetchActions.js

```diff
@@ -1,6 +1,10 @@
 import api from './api';
 import types from './types';
 
+import {
+  updateUsers,
+} from './userActions';
```

Comment:
Wondering why this `import` lives in {} and spans multiple lines as opposed to syntax of imports above?

Reply:
In ES6 imports, there's an `export default` which would be accessible with a normal module import like you see on lines 1 and 2. There's also a simple `export` which allows you to export multiple functions/objects/etc from one file. Notice how the functions in `userActions` begin with `export`. We need this `{ }` import syntax to select which function from that file we're importing. This spans multiple lines to make commits cleaner, so if I were to add or remove a function from this import, it would be a clean line addition or removal.